src/tui/light_windows.go

```diff
@@ -0,0 +1,132 @@
+//+build windows
+
+package tui
+
+import (
+	"os"
+	"syscall"
+
+	"github.com/junegunn/fzf/src/util"
+	"golang.org/x/sys/windows"
```

Comment:
`go build` fails because this is missing from `go.mod`. `go get 'golang.org/x/sys/windows'` should be enough. I can run fzf with `--height` on Windows 10 after this is resolved. Tested on powershell default terminal.

Reply:
@janlazo - Should I submit a separate pull request with golang/sys updated to the latest version?

Reply:
Sure.